### PR TITLE
Enrollment flow style fixes and some adjustments

### PR DESCRIPTION
**Pull request by GiCharkviani**

### CODE REVIEW

File: ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.html [hunk 1]
```diff
@@ -84,7 +84,7 @@
     </aside>
   </main>
   <footer>
-    <button class="btn_secondary" (click)="closeDialog()">Cancel</button>
-    <button class="btn_primary" [disabled]="addPatientForm.invalid" (click)="registerPatient()">Register</button>
+    <button class="btn_secondary" [ngClass]="{'disappear': uploading}" (click)="closeDialog()">Cancel</button>
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.html:87]: We can not cancel uploading ? 
Is it correct?

File: ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.ts [hunk 2]
```diff
@@ -44,11 +45,16 @@ export class RegisterPatientsModalComponent implements OnInit {
   }
 
   public registerPatient(): void {
-    const {patients} = this.addPatientForm.getRawValue();
-    this.closeDialog(patients);
+    console.log(this.addPatientForm.getRawValue());
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.ts:48]: Do we need it?

File: ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.html (hunk 1 above)
Comment (GiCharkviani) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.html:87]: Yes, right

File: ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.ts (hunk 2 above)
Comment (GiCharkviani) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.ts:48]: It's the only option we can see data flow at this moment. when backend is ready, I will remove that

**Review by alextanlan — DISMISSED** [review]

**Review by alextanlan — APPROVED** [review]

**Pull request merged**